presidio-analyzer/app.py

```diff
@@ -26,7 +30,9 @@ class Server:
     """HTTP Server for calling Presidio Analyzer."""
 
     def __init__(self):
-        self.logger = PresidioLogger(os.environ.get("PRESIDIO_LOGGER"))
+        fileConfig(Path(Path(__file__).parent, LOGGING_CONF_FILE))
+        self.logger = logging.getLogger("presidio-analyzer")
+        self.logger.setLevel(os.environ.get("LOG_LEVEL", self.logger.level))
```

Comment:
Is there an attribute for reading that in the logging conf file?

Reply:
Discussed offline, no quick and easy solution.

Reply:
This is for when the log level is assigned in an env var. This would take precedence over the conf file